langlib/lang.runtime/src/main/ballerina/runtime.bal

```diff
@@ -102,6 +141,12 @@ public type StopHandler function() returns error?;
 # that was passed as an argument; the handler functions will be called
 # after calling `gracefulStop` on all registered listeners,
 # in reverse order of the corresponding calls to `onGracefulStop`.
+# 
+# ```ballerina
+# runtime:StopHandler stopHandler = function() returns error? => ();
+# runtime:onGracefulStop(stopHandler);
```

Comment:
```suggestion
# runtime:onGracefulStop(function() returns error? {});
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39006/commits/0c0e021a6e4f631cf924cdff82704d3514345a25